browser/src/canvas/sections/CommentListSection.ts

```diff
@@ -986,6 +986,21 @@ export class CommentSection extends CanvasSectionObject {
 		this.update();
 	}
 
+	public showInNavigator(comment: Comment) {
+		if (app.map.getDocType() === 'text') {
+			/*
+			 * before this, i tried sending position of the comment i.e. data.id
+			 * but that didn't work for new comments as data.id is just a counter
+			 * which can't be used as position for new comments. it worked for already
+			 * existing comments as they were created based on their position in the
+			 * document.
+			 */
+			var commentStr = comment.sectionProperties.data.html;
+			commentStr = commentStr.replace(/<[/]?div>/g, "");
```

Comment:
Please avoid regexp if possible. I think it doesn't need to work for new comment (just showed after "insert comment") until they are applied.
Only when we really insert the comment (OK button) we have it in the core I think

Reply:
awesome! thank you very much